jury/metrics/chrf/chrf_for_language_generation.py

```diff
@@ -0,0 +1,259 @@
+ChrF and ChrF++ are two MT evaluation metrics. They both use the F-score statistic for character n-gram matches,
+and ChrF++ adds word n-grams as well which correlates more strongly with direct assessment. We use the implementation
+that is already present in sacrebleu.
+The implementation here is slightly different from sacrebleu in terms of the required input format. The length of
+the references and hypotheses lists need to be the same, so you may need to transpose your references compared to
+sacrebleu's required input format. See https://github.com/huggingface/datasets/issues/3154#issuecomment-950746534
+See the README.md file at https://github.com/mjpost/sacreBLEU#chrf--chrf for more information.
+"""
+
+_KWARGS_DESCRIPTION = """
+Produces ChrF(++) scores for hypotheses given reference translations.
+Args:
+    predictions: The system stream (a sequence of segments).
+    references: A list of one or more reference streams (each a sequence of segments).
+    char_order: Character n-gram order.
+    word_order: Word n-gram order. If equals to 2, the metric is referred to as chrF++.
+    beta: Determine the importance of recall w.r.t precision.
+    lowercase: Enable case-insensitivity.
+    whitespace: If `True`, include whitespaces when extracting character n-grams.
+    eps_smoothing: If `True`, applies epsilon smoothing similar
+    to reference chrF++.py, NLTK and Moses implementations. Otherwise,
+    it takes into account effective match order similar to sacreBLEU < 2.0.0.
+Returns:
+    'score': The chrF (chrF++) score,
+    'char_order': The character n-gram order,
+    'word_order': The word n-gram order. If equals to 2, the metric is referred to as chrF++,
+    'beta': Determine the importance of recall w.r.t precision
+Examples:
+    >>> prediction = ["The relationship between Obama and Netanyahu is not exactly friendly."]
+    >>> reference = [["The ties between Obama and Netanyahu are not particularly friendly."]]
+    >>> chrf = jury.load_metric("chrf")
+    >>> results = chrf.compute(predictions=prediction, references=reference)
+    >>> print(results)
+    {'score': 61.576379378113785, 'char_order': 6, 'word_order': 0, 'beta': 2}
+"""
+
+
+@datasets.utils.file_utils.add_start_docstrings(_DESCRIPTION, _KWARGS_DESCRIPTION)
+class CHRFForLanguageGeneration(MetricForLanguageGeneration):
+    def _info(self):
+        if version.parse(scb.__version__) < version.parse("1.4.12"):
+            raise ImportWarning(
+                "To use `sacrebleu`, the module `sacrebleu>=1.4.12` is required, and the current version of `sacrebleu` doesn't match this condition.\n"
+                'You can install it with `pip install "sacrebleu>=1.4.12"`.'
+            )
+        return datasets.MetricInfo(
+            description=_DESCRIPTION,
+            citation=_CITATION,
+            inputs_description=_KWARGS_DESCRIPTION,
+            features=self._default_features,
+            codebase_urls=["https://github.com/jitsi/jiwer/"],
```

Comment:
```suggestion
            codebase_urls=["https://github.com/mjpost/sacreBLEU#chrf--chrf"],
```
I'm guessing the codebase should be sacreBLEU.